compass/ocean/tests/overflow/hydro_vs_nonhydro/visualize.py

```diff
@@ -0,0 +1,67 @@
+import matplotlib
+import matplotlib.pyplot as plt
+from netCDF4 import Dataset
+
+from compass.step import Step
+
+matplotlib.use('Agg')
+
+
+class Visualize(Step):
+    """
+    A step for visualizing a cross-section through the fluid
+    descend down the slope.
+
+    """
+    def __init__(self, test_case):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        test_case : compass.TestCase
+            The test case this step belongs to
+        """
+        super().__init__(test_case=test_case, name='visualize')
+
+        for grid in ['nonhydro', 'hydro']:
+            self.add_input_file(filename=f'output_{grid}.nc',
+                                target=f'../{grid}/output.nc')
+            self.add_input_file(filename=f'init_{grid}.nc',
+                                target=f'../{grid}/init.nc')
+        self.add_output_file('section_overflow.png')
+
+    def run(self):
+        """
+        Run this step of the test case
+        """
+        grids = ['hydro', 'nonhydro']
+        nGrids = len(grids)
+        plt.figure(1, figsize=(12.0, 6.0))
+
+        config = self.config
+
+        section = config['horizontal_grid']
+        nx = section.getint('nx')
+        section = config['visualize']
+        time = section.getint('plotTime')
```

Comment:
```suggestion
        time = section.getint('plot_time')
```
Also, see earlier comment about writing `plot_time` in hours

Reply:
Change made.